python/tvm/meta_schedule/testing/utils.py

```diff
@@ -110,3 +115,46 @@ def initialize_with_tune_context(self, context: "TuneContext") -> None:
 
     def apply(self, trace: Trace, _) -> Optional[Trace]:
         return Trace(trace.insts, {})
+
+
+def apply_manual_schedules(
```

Comment:
Suggestions for better names welcome.

Reply:
maybe `apply_schedule_fn`? manual schedule sounds okay but it could potentially cause some confusion about the definition of being manual, like whether or not its tunable

Reply:
Indeed, the intended usage of this API is for manually written schedules with no tunable knobs. In that sense, I think `apply_schedule_fn` is more ambiguous. 

How about `apply_fixed_schedules`?

Reply:
I'm very bad at naming lol. Let's walk back to `apply_manual_schedule` - it sounds good enough?

Reply:
If we compare `apply_manual_schedule` vs `apply_fixed_schedule`, I actually prefer the latter. I can make such change if you agree.

Reply:
I don’t have strong opinion. Let’s go with fixed schedule then. Thank you for doing this!